### PR TITLE
Clarify the uniqueness of Figure and Plot identifiers

### PR DESCRIPTION
**Pull request by otronarp**

### CODE REVIEW

File: chapters/annotations.tex [hunk 1]
```diff
@@ -114,12 +114,16 @@ \subsection{Annotations for Figures\doublelabel{annotations-for-figures}}
...
 \lstinline!title! of \lstinline!Figure! and \lstinline!Plot!.
 
-The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is
-optional, and is intended for programmatic access.  A \lstinline!Figure!
-\lstinline!identifier! must be unique within the class containing the
-\lstinline!figures! annotation, whereas a \lstinline!Plot!
-\lstinline!identifier! must be unique among the \lstinline!plots! in the
-\lstinline!Figure! annotation.
+The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is an
+optional \lstinline!String! identifier, and is intended to identify the
```
Comment (henrikt-ma) [inline chapters/annotations.tex:118]: Is it necessary to say it's a `String` here, given the pseudo-code record definitions?

Comment (otronarp) [reply chapters/annotations.tex:118]: The confusion was about the identifiers living in the same namespace as everything else, spelling out String here should make that more clear.

Comment (henrikt-ma) [reply chapters/annotations.tex:118]: OK, let's see if it the kind of clarification @HansOlsson was looking for.

Comment (HansOlsson) [reply chapters/annotations.tex:118]: Ok, I think it is fine now.

**Comment by HansOlsson** [comment]
Shall we merge it to MCP/0033?

**Comment by henrikt-ma** [comment]
Yes, it seems like @gkurzbach has nothing more to add.

**Comment by gkurzbach** [comment]
@henrikt-ma you are right.

**Pull request merged**